gradle/googleServices.gradle

```diff
@@ -80,9 +80,7 @@ afterEvaluate {
 def isTesting = getGradle().getStartParameter().getTaskNames().any({ String taskName ->
     def hasProjectTestTask = taskName.contains("$name") &&  ['AndroidTest', 'connectedCheck', 'deviceCheck'].any({ taskName.contains(it)})
 
-    // this is needed for storage test app to build
-    def hasStorageAppAssembleTask = taskName == ":firebase-storage:test-app:assembleRelease"
-    return hasProjectTestTask || hasStorageAppAssembleTask || taskName.contains('connectedCheck') || taskName.contains('deviceCheck') || taskName.contains('AndroidTest')
+    return hasProjectTestTask || taskName.contains('connectedCheck') || taskName.contains('deviceCheck') || taskName.contains('AndroidTest')
```

Comment:
why is no longer needed for storage app?

Reply:
The storage app is disabled in the repo, and wouldn't work without secrets anyway, so I decided to remove this effectively "dead" code.